controller/aks-cluster-config-handler.go

```diff
@@ -294,6 +298,12 @@ func (h *Handler) checkAndUpdate(config *aksv1.AKSClusterConfig) (*aksv1.AKSClus
 		return config, fmt.Errorf("update failed for cluster [%s], status: %s", config.Spec.ClusterName, clusterState)
 	}
 	if clusterState == ClusterStatusInProgress || clusterState == ClusterStatusUpgrading {
+		// If the cluster is in an active state in Rancher but is updating in AKS, then an update was initiated in AKS console.
```

Comment:
Not necessarily from the console, just somewhere else.
```suggestion
		// If the cluster is in an active state in Rancher but is updating in AKS, then an update was initiated outside of rancher, such as through the AKS console.
```

Reply:
Done